src/openforms/prefill/base.py

```diff
@@ -31,4 +31,22 @@ def get_prefill_values(
         When no pre-fill value can be found for a given attribute, you may omit the key
         altogether, or use ``None``.
         """
-        raise NotImplementedError("You must implement the 'get_prefill_values' method.")
+        raise NotImplementedError(
+            "You must implement the 'get_prefill_values' method."
+        )  # pragma: nocover
+
+    def get_co_sign_values(self, identifier: str) -> Tuple[Dict[str, Any], str]:
```

Comment:
This works fine, but I was thinking: DigiD/eHerkenning/eIDAS can be configured to retrieve also the name of the person/company in addition to the BSN/KvK. So maybe that could be useful to avoid this extra code.

Docs for eHerkenning/eIDAS: https://documentation.signicat.nl/connectis-identity-broker/configuring-eids/eherkenning-and-or-eidas/about-eherkenning-eidas/requestedattributes

Reply:
yeah, I'm not feeling to fantastic about this warning though:

> Please take note, however, that the eHerkenning specifications do not guarantee that the extra RequestedAttributes are known for each user, and can thus be returned in the response.

Could be a nice improvement in the future! Would avoid an outgoing call if we already get the data on sign-in.